tests/server/index.js

```diff
@@ -1,24 +1,125 @@
     t.error(err, 'No Error');
     t.equal(res.status, 200, 'Should equal 200');
+    t.equal(res.text.includes(html), true, 'The right startup was found');
+  })
+  supertest(server).get('/startupProfile/sdgadfasd').end((err, res) => {
+    t.equal(res.text.includes(html2), true, 'Recieved 404 page');
     t.end();
-  });
-});
+  })
+})
+
+
+tape('Test for register startup route', (t) => {
+  let expected = {
+    'startup-name': 'FAC',
+    'founder-name': 'Dan',
+    'coFounder-name': '',
+    'description': 'Coding bootcamp for everyone',
+    Select: 'Software',
+    'funding-stage': 'Seed-Funded',
+    'startup-email': 'mario91sss@gmail.com',
+    'startup-website': 'foundersandcoders.com'
+  }
+  supertest(server).post('/registerStartup')
+  .send(expected)
+  .end((err, res) => {
+    t.error(err, 'No Error');
+    t.end();
+  })
+})
+
+tape('Test the submit message page', (t) => {
+  let html = 'Thank you!'
+  supertest(server).get('/submitMessage').end((err, res) => {
+    t.error(err, 'No Error');
+    t.equal(res.text.includes(html), true, 'Should have the html');
+    t.end();
+  })
+})
+
+tape('Test for results', (t) => {
+  supertest(server).get('/search?search=&sector=').end((err, res) => {
+    t.error(err, 'No Error');
+    t.equal(res.status, 200, 'Should equal 200');
+  })
+  supertest(server).get('/search?search=&sector=Health').end((err, res) => {
+    t.error(err, 'No Error');
+    t.equal(res.status, 200, 'Should equal 200');
+  })
+  supertest(server).get('/search?search=MyCity&Submit=Submit&sector=cities').end((err, res) => {
+    t.error(err, 'No Error');
+    t.equal(res.status, 200, 'Should equal 200');
+  })
+  supertest(server).get('/search?search=MyCIty&Submit=Submit').end((err, res) => {
+    t.error(err, 'No Error');
+    t.equal(res.status, 200, 'Should equal 200');
+  })
+  supertest(server).get('/search?search=MyCity&Submit=Submit&stage=bootstrap').end((err, res) => {
+    t.error(err, 'No Error');
+    t.equal(res.status, 200, 'Should equal 200');
+  })
+  supertest(server).get('/search?search=MyCity&Submit=Submit&sector=Cities&stage=Bootstrap').end((err, res) => {
+    t.error(err, 'No Error');
+    t.equal(res.status, 200, 'Should equal 200');
+  })
+  supertest(server).get('/search?search=&Submit=Submit&sector=Cities&stage=Bootstrap').end((err, res) => {
+    t.error(err, 'No Error');
+    t.equal(res.status, 200, 'Should equal 200');
+  })
+  supertest(server).get('/search?search=&Submit=Submit&stage=Idea-Phase').end((err, res) => {
+    t.error(err, 'No Error');
+    t.equal(res.status, 200, 'Should equal 200');
+  })
+  let html = '<h1 class="f1 fw3 text-granite-gray">Startups</h1>\n';
+  supertest(server).get('/search').end((err, res) => {
+    t.error(err, 'No Error');
+    t.equal(res.status, 200, 'Should equal 200');
+    t.equal(res.text.includes(html), true, 'Should render the results page');
+  })
+  supertest(server).get('/search?search=&sector=1').end((err, res) => {
```

Comment:
but what if at some point of the future i added the startup then that test would fail,
i dont think we need a test for how many results we are getting